corehq/apps/app_manager/models.py

```diff
@@ -4467,8 +4468,35 @@ def make_build(self, comment=None, user_id=None):
         assert copy._id
         prune_auto_generated_builds.delay(self.domain, self._id)
 
+        self.check_build_dependencies(new_build=copy)
+
         return copy
 
+    def check_build_dependencies(self, new_build):
+        """
+        Reports whether the app dependencies have been removed.
```

Comment:
looks like the documentation needs to be updated.
And possibly the function name as well.

Reply:
Good catch

Reply:
https://github.com/dimagi/commcare-hq/pull/35455/commits/08b1425cb8e8c3919874bbcc836e43d64eaa7669